### ISSUE TITLE
Ik kan geen headers meesturen?

### ISSUE THREAD

**Issue by mwaneykm** [het-keukenmagazijn/passport-authenticator#2]
Als ik ids meegeef als 2e of 3e argument `$this->_authenticator->post('/api/deliveries/read', ['Ids' => 'Ids:' . http_build_query($productIds), 'Montage' => 0]);`

dan worden ze buiten de header geplaatst

`  "headers" => array:3 [▶]
  "Ids" => "Ids:0=2020343&1=2011265&2=2020545&3=2042587&4=2009713&5=2030557&6=2009082&7=2016341&8=2020357&9=2042416&10=2015550&11=2023432&12=2042560&13=2023746&14=2007933"
  "Montage" => 0`

**Comment by ghost** [comment]
You're using the $datatoSend argument instead of the $extraHeaders argument,
try this code instead:
```php
$this->_authenticator->post('/api/deliveries/read', [], ['Ids' => 'Ids:' . http_build_query($productIds), 'Montage' => 0]);
```

**Comment by ghost** [comment]
There was a bug where the headers weren't merged properly, fixed in e617b2e.
